### ISSUE TITLE
App will sometimes crash near startup

### ISSUE THREAD

**Issue by clay53** [lbryio/lbry-android#1062]
## The Issue
I think it's being caused when the background process is loading as I haven't experienced a crash after that has finished.
Error:
```
2020-12-12 20:23:20.628 24040-24190/io.lbry.browser E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #3
    Process: io.lbry.browser, PID: 24040
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$3.done(AsyncTask.java:353)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
        at java.util.concurrent.FutureTask.run(FutureTask.java:271)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
        at java.lang.Thread.run(Thread.java:764)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int org.json.JSONArray.length()' on a null object reference
        at io.lbry.browser.tasks.wallet.LoadSharedUserStateTask.isNotificationsDisabledForSubUrl(LoadSharedUserStateTask.java:134)
        at io.lbry.browser.tasks.wallet.LoadSharedUserStateTask.doInBackground(LoadSharedUserStateTask.java:85)
        at io.lbry.browser.tasks.wallet.LoadSharedUserStateTask.doInBackground(LoadSharedUserStateTask.java:36)
        at android.os.AsyncTask$2.call(AsyncTask.java:333)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636) 
        at java.lang.Thread.run(Thread.java:764)
```

### Steps to reproduce
1. Pull from the latest on master 0647deb06cb3bc3e8a87dfa6453efa6dc4411c58
2. Set up development environment
3. Build/run/install app
4. Wait for background task to load and it may crash during that time

### Expected behaviour
Should not crash

### Actual behaviour
Crashed

## System Configuration
- Device: Samsung Galaxy S7
- Android version: 8.1

**Comment by clay53** [comment]
Some more testing, seems to not crash until I try to do anything on the app after wallet and stuff loads. Background process itself doesn't crash but the interface does.

**Comment by akinwale** [comment]
Fixed in 0.16.11 which is now live.
